okapi-core/src/main/raml/okapi.raml

```diff
@@ -1128,6 +1128,31 @@ types:
         body:
           text/plain:
 
+/_/proxy/import/modules:
+  description: Import modules
+  post:
+    description: Import list modules.
```

Comment:
The descriptions of the related endpoints should link to each other, be similar and verbose.

```
/_/proxy/modules:
  post:
    description: Announce one new module to the proxy. Once successful a module
      can be selected for a specific tenant. To announce a list of modules use
      /_/proxy/import/modules.
```

```
/_/proxy/import/modules:
  post:
    description: Announce a list of new modules to the proxy. Once successful a module
      can be selected for a specific tenant. To announce only one module you may use
      /_/proxy/modules.
```

Reply:
ok